pkg/aws/ec2_test.go

```diff
@@ -928,6 +952,7 @@ func TestAssertEC2VolumeType_MatchWithThroughputMultipleDevices(t *testing.T) {
 
 func TestAssertEC2VolumeType_MatchWithIOPSMultipleDevices(t *testing.T) {
 	// Setup
+	// t.Parallel()
```

Comment:
**Suggestion:** Would you please uncomment this, unless there's a reason this test can't be run in parallel, in which case would you please remove the commented code?

Reply:
addressed